src/operator/controllers/intents_reconcilers/otterizecloud/status_report.go

```diff
@@ -0,0 +1,38 @@
+package otterizecloud
+
+import (
+	"context"
+	"github.com/otterize/intents-operator/src/shared/otterizecloud/graphqlclient"
+	"github.com/otterize/intents-operator/src/shared/otterizecloud/otterizecloudclient"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+)
+
+func PeriodicallyReportConnectionToCloud(client CloudClient) {
+	interval := viper.GetInt(otterizecloudclient.ComponentReportIntervalKey)
+	go func() {
```

Comment:
You can `go runPeriodicReportConnection(interval, client)` directly, without calling it within another function.

Reply:
I know. I just hate to read it this way, specially when the actual function run inside a select statement inside a for loop.

Reply:
That's what makes it a Start though - it starts a goroutine that keeps running forever